query/sources/impl/src/test/groovy/org/codice/ddf/admin/sources/wfs/discover/DiscoverWfsSourcesTest.groovy

```diff
@@ -187,4 +184,25 @@ class DiscoverWfsSourcesTest extends Specification {
         } == 1
         report.messages()*.getPath() == [URL_FIELD_PATH]
     }
+
+    def prepareOpenSearchSourceUtils(int statusCode, String responseBody, boolean endpointIsReachable) {
+        def requestUtils = new TestRequestUtils(createMockFactory(statusCode, responseBody), endpointIsReachable)
+        def wfsUtils = new WfsSourceUtils()
+        wfsUtils.setRequestUtils(requestUtils)
+        return wfsUtils
+    }
+
+    def createMockFactory(int statusCode, String responseBody) {
```

Comment:
We should trying making a class that is extensible to inherit these methods. Definitely would leave a comment saying why the class was made.